torch/_functorch/_aot_autograd/collect_metadata_analysis.py

```diff
@@ -335,7 +336,7 @@ def inner(*flat_args):
             )
             curr_storage = (
                 None
-                if not isinstance(o, torch.Tensor)
+                if not isinstance(o, torch.Tensor) or is_batchedtensor(o)
```

Comment:
TODO: I'm not sure what's going on in this file

Reply:
We don't actually need those changes. I'll remove them.